testdata/templates/foo.yaml

```diff
@@ -6,6 +6,7 @@ routingKey: rk
 routingDelegate: rd
 thrift: foo.thrift
 timeout: 4.5s
+disableThriftEnvelope: false
```

Comment:
Isn't `disableThriftEnvelope: false` the default? Possibly add a separate YAML with it set to true?